scripts/forseti-client/forseti_environment.sh.tpl

```diff
@@ -0,0 +1,2 @@
+export FORSETI_HOME=${forseti_home}
```

Comment:
It looks like the `.tpl` suffix leaked through to the `forseti_environment.sh` file:

```
ubuntu@forseti-client-vm-df4f4f72:~$ cat /etc/profile.d/forseti_environment.sh
export FORSETI_HOME=/home/ubuntu/forseti-security
export FORSETI_CLIENT_CONFIG=/home/ubuntu/forseti-security/configs/forseti_conf_client.yaml.tpl
```

In addition it looks like this path is not correct:

```
ubuntu@forseti-client-vm-df4f4f72:~$ ls /home/ubuntu/forseti-security/configs/
client  logging  server
```

However the file is named correctly:

```
ubuntu@forseti-client-vm-df4f4f72:~$ ls /home/ubuntu/forseti-security/configs/client/
forseti_conf_client.yaml  forseti_conf_client.yaml.in  forseti_conf_client.yaml.sample
```